shared/utils/parseDocumentSlug.js

```diff
@@ -1,12 +1,16 @@
 // @flow
 
-export function parseDocumentSlugFromUrl(url: string) {
+export default function parseDocumentSlugFromUrl(url: string) {
   let parsed;
-  try {
-    parsed = new URL(url);
-  } catch (err) {
-    return;
+  if (url[0] === "/") {
+    parsed = url;
+  } else {
+    try {
+      parsed = new URL(url).pathname;
+    } catch (err) {
+      return;
+    }
```

Comment:
allow for parsing of paths as well as fully qualified URLs

Reply:
Looks good, this util is deserving of a test not sure why it doesn't have one already